api-scripts/patron/placing-holds/jmeter-supported-data/delete_requests.sql

```diff
@@ -0,0 +1,3 @@
405c3f1f987','d09ccd34-1b54-4587-b5df-0cf90fb7ac86','65ef75cb-0b64-4c09-b7c4-304201869f0c','2241f825-b44d-4163-ae52-5eaffadf9242','e1f886fc-5630-439d-81ac-e455776f9145','558568ff-777b-4082-a942-53a93c0fed27','cd9bf5f5-3919-4b39-87c7-35eb8df2c22b','3fbf48ef-bb63-492d-8a7c-a6ba1514869c','b48e2107-8b7d-4302-8914-3f91e35ba837','89b75729-62a1-448d-bb25-c4aefa18a5a0','14b899a3-ab6f-4ab9-8962-38735257511d','9d8a4e3c-2c2c-49e2-966b-ba51f12335d7','a017f7b5-2826-44ad-b4ca-f125a5593a86','68571d4c-f932-4b66-aa0f-db8207a93029','55482842-ea98-41f8-879b-9cb3c59be3c2','34011246-e681-4351-9479-0a6afa552a62','0dfc6154-f74d-4d71-b146-e88c060e5986','439aa15f-0768-4cd0-bfcd-6160eea03e76','799792df-5f6b-493d-98dd-31c8abd032e2','c9d1565d-bd65-4214-8041-d17f9f36ccc3','293d991f-696a-488f-bf31-cdde248e9632','a35a1b75-5d30-40c3-bf25-7a704788e3c7','80a65896-269f-47f9-9292-121a45f7b38d','76d9a28a-7f5d-4b1c-b66c-e2d911a0440a','3b10f26b-1e49-49c3-be89-da680ec15c45','92584da5-72d4-41a9-8077-7eff928bd428','df2140de-f584-477d-aec1-e1dab1d86734','b531b9c0-e467-4507-aacb-0e3eedd3b2df','69303f6e-7e59-4733-b7c2-51e25d442148','eb3a3388-6bb9-4a20-9c03-e0116d2da0f4','7f624fbe-9a01-435d-abff-8a2e8f42c053','c9b2f2b8-8a6b-4248-8c0e-c369ed201831','aa542d04-7cf9-4f05-9482-5cf614bbbaa9','df4c5d14-ee71-4be3-9e9c-355c7ac98eda','0b6cf8e2-9961-425f-a524-2bb50e1bc2a4','ce2403ef-0688-4b6b-a86c-b8de354defa0','75174ea9-610e-4fc6-8ffa-4ab10d116b24','ea346fcb-605c-4829-acfd-dd9318c893a5','93d194d8-82ec-4c61-bdb5-61d7ac28ffdb','78fe462c-8175-4196-8973-fcabcce7195a','9e2cdab0-1762-4b28-b19a-19e96452e8d9','b8c4fb32-1834-44e6-8dcc-427946732655','0778e722-e001-4d2b-a675-f2a048245a56','a5b8c4a7-7332-482d-968b-54bf64716c91','91ad829e-1ae6-4392-bf5b-e907f19227e5','8442f39f-0d2a-4f5d-934e-59562e3d3d6e','3f8aa9f1-bb08-4d00-a49a-d87c1a0346b8','d11c7ab2-642b-451b-8993-6dd85e615e32','28cf347e-2cbd-46de-83fe-7e8a44ca51b9','8f1eacbe-5201-40ef-a558-5a40b08b4107','cac8cf6f-6ebc-4bb5-adc9-289e7e84936a','c73fdfdc-3037-4553-9d90-5c7d4f25a763','14d6471f-9c69-4f6a-a8d0-2103f43d560a','062c9d70-0454-4e23-80e9-6106f90e714e','de9f1dcd-234f-4a2f-99af-c72be5e65b58','b5598711-bab9-47ff-81a6-b8f2e9861fcd','9ba62779-6636-4d6b-b23a-198a05a2a60f','384355ea-adc2-497f-995d-ff6a035294f4','42fdacfb-5d94-4aa1-8ee6-0b1a9a2604f0','c75dcd61-53ae-4dce-98f7-b9b87b3b0fcc','d717c90c-3657-4489-9ad6-9fd36807f28c','3db3c3db-4cd7-4376-bdc6-0f40eb285be2','12d6e7e6-5656-4fe4-9c23-9580628bd7d5','bc559e99-924a-43c3-b8bc-78cb04e33eaf','c8935eec-8262-4ef7-a5d0-af3a59029c13','34ddf4fb-6c05-49c7-bd13-ac1219e75407','357ec97a-36d9-4e87-a51b-e8db46bbdacd','133a6171-d72c-49e2-b2a3-6b9e1d28e852','37b34059-8a52-4467-be3f-9c94cec40d55','2574acf3-953e-4187-a16b-97c06681ce4d','e4b406b0-34a6-47d6-b376-5253d6e1dc59','8bd9c46a-4eb9-4777-b2bf-48640ce1ca76','8e73031e-d614-44c0-9d01-16076684bc4e','502ee9c8-fa21-4ad0-9485-366e23e13f05','f1cd5408-aa23-4091-89da-7e7fe14286a3','50b06007-66d0-46bf-8270-8530dd312920','5c53294d-b4ad-4d94-a581-375b45208460','25879b50-ccbc-4eca-a2f
```

Comment:
To avoid hardcoding long list of uuids, can we update query to
```DELETE FROM fs09000000_mod_circulation_storage.request  as req 
	WHERE req.jsonb->>'requesterId' IN (
SELECT req.jsonb->>'requesterId'
	FROM fs09000000_mod_circulation_storage.request as req GROUP BY jsonb->>'requesterId');```

Reply:
We need the list of users' ids which we've created the requests, not just first 10,000 ids.
If we use the query which we used to get users ids, the user's ids will be different as they already have requests.
That why I write hardcoding long list of uuids